README.md

```diff
@@ -1,59 +1,97 @@
 # Thali Cordova Plugin
 
-<Work on progress, not near release quality !!>
-## Goals
-The Thali Cordova Plugin is an easy-to-use Cordova plugin for building peer-to-peer (P2P) networking
-apps on Android and iOS. 
+This project is a work in progress and not yet production-level quality.
 
-The Thali Cordova Plugin is layered on the JXcore Cordova plugin, which uses JXcore to allow one to build
-mobile applicatons in JavaScript for Node.JS.  
+The Thali Cordova Plugin is a [Cordova](http://cordova.apache.org/) plugin for building peer-to-peer (P2P) networking apps on Android and iOS.
+
+The Thali Cordova Plugin is layered on the [JXcore Cordova plugin](https://github.com/jxcore/jxcore-cordova), which uses [JXcore](http://jxcore.com/home/) to allow one to build mobile applications in JavaScript for Node.JS.  
 
 ## Prerequisites
 
-* For Android, [Android Studio](http://developer.android.com/sdk/index.html)
-* For iOS, [Xcode 6](https://developer.apple.com/xcode/), or later
+### Android
+
+Download [Android Studio](http://developer.android.com/sdk/index.html)
+
+Make sure to set your `ANDROID_HOME` environment variable:
+
+Mac OS X (put in your `~/.bash_profile` file):
+```
+export ANDROID_HOME=~/Library/Android/sdk
+export PATH=${PATH}:$ANDROID_HOME/tools:$ANDROID_HOME/platform-tools
+```
+
+Linux (put in your `~/.bashrc` file):
+```
+export ANDROID_HOME=/<installation location>/Android/sdk
+export PATH=${PATH}:$ANDROID_HOME/tools:$ANDROID_HOME/platform-tools
+```
+
+Windows:
+```
+set ANDROID_HOME=C:\<installation location>\Android\sdk
+set PATH=%PATH%;%ANDROID_HOME%\tools;%ANDROID_HOME%\platform-tools
+```
+
+### iOS
+
+Download [Xcode 6](https://developer.apple.com/xcode/), or later.
 
 ## Getting Started
 
-### Install latest Node.JS
-Follow the instructions at [https://nodejs.org/](https://nodejs.org/). When you're done, check that the
-installation worked: 
+### Install latest JXCore
+
+Follow the instructions at [http://jxcore.com/downloads/](http://jxcore.com/downloads/). When you're done, check that the
+installation worked:
 ```
-~>   node -v
-v0.12.2
-~>   npm -v
-2.7.4
+$ jx -jxv
+v Beta-0.3.0.3
 ```
 
 ### Install Cordova
-(Check the [Android Platform Guide](https://cordova.apache.org/docs/en/4.0.0/guide_platforms_android_index.md.html#Android%20Platform%20Guide) 
+
+(Check the [Android Platform Guide](https://cordova.apache.org/docs/en/4.0.0/guide_platforms_android_index.md.html#Android%20Platform%20Guide)
 and [iOS Platform Guide](https://cordova.apache.org/docs/en/4.0.0/guide_platforms_ios_index.md.html#iOS%20Platform%20Guide) for detailed instructions.)
+
 ```
-~> sudo npm install -g cordova
+$ sudo jx install -g cordova
 ```
 
 ### Create a Cordova project
+
 ```
-~/Code> cordova create ThaliTest com.test.thalitest ThaliTest
+$ cordova create ThaliTest com.test.thalitest ThaliTest
 ```
 
 ### Android Requirements
```

Comment:
Shouldn't these instructions be moved to a dev section since users don't need to do this anymore? We only need this when we are building and updating maven?